.editorconfig

```diff
@@ -208,3 +208,7 @@ csharp_preserve_single_line_statements = true
 
 # warning RS0037: PublicAPI.txt is missing '#nullable enable'
 dotnet_diagnostic.RS0037.severity = none
+
+[src/CodeStyle/**.{cs,vb}]
+# warning RS0005: Do not use generic CodeAction.Create to create CodeAction
```

Comment:
This was specifically done to ensure telemetry is logged for our code actions. Why is this change required? All IDE code fixers already respect RS0005.

Reply:
> Why is this change required?

All the analyzers were disabled prior to this change. I am enabling all of them except this one.

> All IDE code fixers already respect RS0005.

The ones in the code style layer do not.

Reply:
Sounds good. I think if we decide to disable this rule in Features layer as well, we should confirm in the design meeting that telemetry from these actions is not required.